ichnaea/scripts/datamap.py

```diff
@@ -59,7 +59,7 @@ def export_file(filename, tablename, _db=None, _session=None):
         )
     )
 
-    db = configure_db("ro", _db=_db)
+    db = configure_db("ro", _db=_db, pool=False)
```

Comment:
Now that we have a `pool` argument, we can pass it in here and make sure it's not using a pool.